### PR TITLE
Bug/case list missing contact details

### PR DESCRIPTION
**Pull request by madetech-jeffpinkham**
Replace primary contact name being pulled from contacts table with it being pulled from Members table

### CODE REVIEW

File: LBHTenancyAPI/Gateways/UhTenanciesGateway.cs [hunk 1]
```diff
@@ -155,23 +152,24 @@ public Tenancy GetTenancyForRef(string tenancyRef)
...
-                    "LEFT JOIN property " +
-                    "ON property.prop_ref = tenagree.prop_ref " +
-                    "WHERE tenagree.tag_ref = @tRef " +
-                    "ORDER BY arag.arag_startdate DESC",
+                    @"
+                    SELECT TOP 1
+                    tenagree.tag_ref as TenancyRef, 
+                    tenagree.cur_bal as CurrentBalance, 
+                    tenagree.tenure as Tenure, 
+                    RTRIM(LTRIM(member.forename)) + ' ' + RTRIM(LTRIM(member.surname)) as PrimaryContactName,
```
Comment (StevenLeighton21) [inline LBHTenancyAPI/Gateways/UhTenanciesGateway.cs]: You moved the trim to the SQL from the domain object? I have no preference either way, interested in the reasoning :)

Comment (madetech-jeffpinkham) [reply LBHTenancyAPI/Gateways/UhTenanciesGateway.cs]: No particular reason I wasn't aware there was trimming going on with the TenancyListItem and some of the member.forename and member.surname, would have a lot of whitespace on them. Due to it the TenancyListItem.PrimaryContactName being {firstname} + {lastname} I trimmed at the SQL level.

File: LBHTenancyAPI/Gateways/UhTenanciesGateway.cs [hunk 2]
```diff
@@ -155,23 +152,24 @@ public Tenancy GetTenancyForRef(string tenancyRef)
...
+                    tenagree.tag_ref as TenancyRef, 
+                    tenagree.cur_bal as CurrentBalance, 
+                    tenagree.tenure as Tenure, 
+                    RTRIM(LTRIM(member.forename)) + ' ' + RTRIM(LTRIM(member.surname)) as PrimaryContactName, 
+                    property.address1 as PrimaryContactLongAddress, 
+                    property.post_code as PrimaryContactPostcode 
+                    FROM tenagree WITH(NOLOCK)
+                    LEFT JOIN arag WITH(NOLOCK)
+                    ON arag.tag_ref = tenagree.tag_ref 
+
```
Comment (StevenLeighton21) [inline LBHTenancyAPI/Gateways/UhTenanciesGateway.cs]: ooooh rogue extra line ooooooh

File: LBHTenancyAPITest/Helpers/Data/TestDataHelper.cs [hunk 3]
```diff
@@ -39,11 +39,23 @@ public static void InsertMember(Member member, SqlConnection db)
 
         public static void InsertProperty(Property property, SqlConnection db)
         {
-            var commandText = "INSERT INTO property(short_address, address1, prop_ref, post_code) VALUES(@short_address, @address1, @prop_ref, @post_code);";
+
```
Comment (StevenLeighton21) [inline LBHTenancyAPITest/Helpers/Data/TestDataHelper.cs]: ooooh rogue whitespace oooooos

**Comment by madetech-jeffpinkham** [comment]
Sweet thanks

**Pull request merged**